h/accounts/views.py

```diff
@@ -315,14 +315,14 @@ def _reset_password(self, user, password):
 class RegisterController(object):
 
     def __init__(self, request):
-        tos_link = ('<a href="/terms-of-service">' +
+        tos_link = ('<a class="link" href="/terms-of-service">' +
                     _('Terms of Service') +
                     '</a>')
-        cg_link = ('<a href="/community-guidelines">' +
+        cg_link = ('<a class="link" href="/community-guidelines">' +
                    _('Community Guidelines') +
                    '</a>')
         form_footer = _(
-            'You are agreeing to be bound by our {tos_link} and '
+            'You are agreeing to our {tos_link} and '
```

Comment:
There is a small remaining issue that the underline below links overlaps the line below slightly when the message wraps onto multiple lines. I'll come back to that it is very minor I want to run something by Conor.